versioned_docs/version-2.0/self_hosting/configuration/blob_storage.mdx

```diff
@@ -111,3 +112,74 @@ If using an access key and secret, you can also provide an existing Kubernetes s
 This is recommended over providing the access key and secret key directly in your config.
 
 :::
+
+## TTL Configuration
+
+If using the [TTL](/self_hosting/configuration/ttl) feature with LangSmith, you'll also have to configure TTL rules for
+your blob storage. Trace information stored on blob storage is stored on a particular prefix path, which determines the TTL for the
+data. As traces are upgrades, they are place in the upgrade prefix path as well to ensure that the data is upgraded in
```

Comment:
I'm confused what you mean here

Reply:
Do you mean something like:

```
When a trace's retention is extended, its corresponding blob storage path changes to ensure that it has a path that matches the new extended retention. 
```

Reply:
Update: 

`When a trace's retention is extended, its corresponding blob storage path changes to ensure that it matches the new extended retention.`